wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/ExpressionElement.scala

```diff
@@ -17,6 +17,18 @@ object ExpressionElement {
     */
   final case class StringPlaceholder(expr: ExpressionElement) extends StringPiece
 
+  sealed trait StringEscapeSequence extends StringPiece {
+    def unescape: String
+  }
+  case object NewlineEscape extends StringEscapeSequence { override val unescape: String = System.lineSeparator }
```

Comment:
I would either change `parseEscapeSequence()` to also use something like `case System.lineSeparator`, or change this to be `override val unescape = "\\n"`.

Otherwise on some systems the values may not match. Even if we don't support those systems, I still prefer consistency.

Reply:
They're actually subtly different - `parseEscapeSequence` is looking at what escape sequence was in the WDL (eg if my script has `String s = "\n"`) - this line is specifying what value to replace the escape sequence with in the resulting scala `String` value